Hyperlinking client-go and clarifying a sentence (#44886)
kubernetes/website: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

Once created, a CertificateSigningRequest must be approved before it can be signed.
Depending on the signer selected, a CertificateSigningRequest may be automatically approved
by a {{< glossary_tooltip text="controller" term_id="controller" >}}.

> Otherwise, a CertificateSigningRequest must be manually approved either via the REST API - the most common methods for making these API requests include kubectl (by running `kubectl certificate approve`) and the Kubernetes go client library [client-go](https://github.com/kubernetes/client-go). Likewise, a CertificateSigningRequest may also be denied,

Reviewer: ```suggestion
Otherwise, a CertificateSigningRequest must be manually approved either via the REST API - the most common
methods for making these API requests include kubectl (by running `kubectl certificate approve`)
and the Kubernetes go client library [client-go](https://github.com/kubernetes/client-go). Likewise, a CertificateSigningRequest may also be denied,
```

Reviewer: Please wrap the lines and follow contributor [styleguide](https://kubernetes.io/docs/contribute/style/style-guide/) for contribution rules.

Reviewer: This change doesnt add any improvement in the content. However it looks like the sentence is modified to say the exactly same thing.

Once created, a CertificateSigningRequest must be approved before it can be signed.
Depending on the signer selected, a CertificateSigningRequest may be automatically approved
by a {{< glossary_tooltip text="controller" term_id="controller" >}}.

> Otherwise, a CertificateSigningRequest must be manually approved via the REST API - the most common

Reviewer: the hyperlink to client-go is great but otherwise the previous sentence looked and sounded much better. Is there anything more you want to add here?

> Otherwise, a CertificateSigningRequest must be manually approved via the REST API - the most common
> methods for making these API requests include kubectl (by running `kubectl certificate approve`)
> and the Kubernetes go client library [client-go](https://github.com/kubernetes/client-go). Likewise, a CertificateSigningRequest may also be denied,

Reviewer: ```suggestion
Otherwise, a CertificateSigningRequest must be manually approved
either via the kubectl (by running `kubectl certificate approve`)
or via REST API i.e. the Kubernetes go client library [client-go](https://github.com/kubernetes/client-go). Likewise, a CertificateSigningRequest may also be denied,
```
Something along these lines. Also, **note** here in previous text REST API (or client-go) are mentioned for one method and kubectl for second. I understand your context but let's not confuse the user.
